GoodThingsComein3s/Views/RestaurantCollectionViewCell.m

```diff
@@ -0,0 +1,28 @@
+//
+//  RestaurantCollectionViewCell.m
+//  GoodThingsComein3s
+//
+//  Created by Ruhee Rajwani on 7/21/22.
+//
+
+#import "RestaurantCollectionViewCell.h"
+#import <Parse/Parse.h>
+
+@implementation RestaurantCollectionViewCell
+
+-(void) setRestaurant:(PFObject *)restaurant{
```

Comment:
Feedback regarding spacing: Seen this in a few other places but methods should look like 

`- (void)setRestaurant:(PFObject *)restaurant {`